reo/nested_inputs.py

```diff
@@ -1658,7 +1708,11 @@ def list_of_dict(input):
           "min": 0.0,
           "max": 1.0,
           "description": "Knockdown factor on absorption chiller COP based on the CHP prime_mover not being able to produce as high of temp/pressure hot water/steam"
-        }
+        },
+        "can_supply_st": {
```

Comment:
Why not spell out "steam" in this input name? I think that we should to make it clearer.

Reply:
st actually = "steam turbine", so it's kind of an acronym. However, your point is still valid and I think it's not clear enough - I was just being concise but at the expense of clarity. I'll rename to `can_supply_steam_turbine`.